client/app/telecom/telephony/service/contact/translations/Messages_fr_FR.xml

```diff
@@ -26,6 +26,7 @@
     <translation id="telephony_service_contact_urbandistrict">Arrondissement (pour Paris, Lyon et Marseille)</translation>
     <translation id="telephony_service_contact_waynumber">Numéro de voie</translation>
     <translation id="telephony_service_contact_wayname">Nom de la voie</translation>
+    <translation id="telephony_service_contact_addressextra">Complément d'adresse</translation>
```

Comment:
`'` => `’`

Reply:
same keychar as all others `'` in the file, for example : "Publication dans l'annuaire inversé", should i replace all others ? Or do i leave it as it ?